libvast/src/system/posix_filesystem.cpp

```diff
@@ -72,7 +73,8 @@ posix_filesystem(filesystem_actor::stateful_pointer<posix_filesystem_state> self
         ++self->state.stats.checks.successful;
       } else {
         ++self->state.stats.checks.failed;
-        return caf::make_error(ec::no_such_file, err.message());
+        return caf::make_error(ec::no_such_file,
+                               fmt::format("{} no such file: {}", *self, path));
```

Comment:
Displaying `err.message()` might still be valuable, because it shows which error was returned by the OS filesystem.

Reply:
ah, thanks. I was assuming that it's always "file not found", so I thought it's useless.

Will fix